app/columns.tsx

```diff
@@ -607,6 +607,30 @@ export const completedExperimentColumns: ColumnDef<RecipeOrBranchInfo>[] = [
       );
     },
   },
+  {
+    accessorKey: "metrics",
+    header: () => "Metrics",
+    cell: (props: any) => {
```

Comment:
How about hoisting this into `experimentUtils.ts` so that if we want to add or remove experiments from the list, we don't inadvertently miss a place that we need to edit...